x/name/types/codec.go

```diff
@@ -18,11 +17,6 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 		&MsgCreateRootNameRequest{},
 	)
 
-	registry.RegisterImplementations(
-		(*govtypesv1beta1.Content)(nil),
-		&CreateRootNameProposal{},
-	)
-
```

Comment:
There's a very good chance we need to keep this registration around forever.

My fear is that things will blow up when something (e.g. a gov proposals query) tries to read an old proposal with one of these in it.